static/js/hotkey.js

```diff
@@ -326,10 +326,10 @@ function process_hotkey(e) {
         if ((event_name === 'up_arrow' || event_name === 'down_arrow') && focus_in_empty_compose()) {
             compose.cancel();
             // don't return, as we still want it to be picked up by the code below
-        } else if (event_name === "page_up") {
+        } else if (event_name === "page_up" && e.type === "keydown") {
```

Comment:
We should figure out how to do this hack inside of get_hotkey_from_event(), rather than having this here.  It may lead to a more general fix, and it also localizes all the complications due to the strange ways that browsers make keyboard events.